app/views.py

```diff
@@ -74,6 +73,11 @@ def faq():
     """Renders the faq page."""
     return render_template('faq.html', title=' | FAQ')
 
+@app.route('/progressreport')
```

Comment:
Most likely the progress report would have routing for every knewbieID, so this function would need to reflect that.

Reply:
I already have the app route, no?

Reply:
Nope, each user should have a unique link to access their progress report. `/progressreport` only goes to the user's own progress report.

Reply:
ahh yes, we can do that when we get round to issue #12